FlowCal/io.py

```diff
@@ -1095,25 +1097,29 @@ def amplifier_gain(self, channels=None):
         else:
             return self._amplifier_gain[channels]
 
-    def domain(self, channels=None):
+    def range(self, channels=None):
         """
-        Get the domain of the specified channel(s).
+        Get the range of the specified channel(s).
 
-        The domain is inferred from the $PnR parameter, as
-        ``np.arange($PnR)``. The domain should be transformed along with
-        the data when passed through a transformation function.
+        The range is inferred from the $PnR parameter as ``[0, $PnR - 1]``.
```

Comment:
Actually, I ran into an issue here. Assume that the range is `[0, $PnR]` and the resolution is `$PnR`. The current implementation of `hist_bins()` would then generate, by default, `$PnR` bins with weird spacing different than one. We could live with these weird bins, or we could make the resolution `$PnR + 1` with floating points, or we could have the range always be `[0, $PnR - 1]`.

I choose the latter. So far we have two independent examples of floating point files, one of which blatantly disregards the standard's description of `$PnR` (i.e. has obviously a higher resolution ADC). In this file, events seem to be in `[0, $PnR]`, although there are negative numbers. Our other, more standard example *does* have channels with events in `[0, $PnR - 1]`, while using an ADC that clearly has the resolution specified by `PnR`, as I showed you before.

Ultimately, the precise range for floating points may not be that important, given that the standard specifies that the file may contain numbers outside it. I think what I chose is the best option, but I don't think it's better by much.